rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerRestClient.java

```diff
@@ -74,17 +84,31 @@
 
     private String restEndpointURL;
 
-    private ClientHttpEngine httpEngine;
-
     private ResteasyProviderFactory providerFactory;
 
+    private static ClientHttpEngine httpEngine;
+
+    private static SSLContext sslContext;
+
     public SchedulerRestClient(String restEndpointURL) {
         this(restEndpointURL, null);
     }
 
     public SchedulerRestClient(String restEndpointURL, ClientHttpEngine httpEngine) {
-        this.restEndpointURL = restEndpointURL;
+
+        if (sslContext == null) {
+            setBlindTrustSSLContext();
```

Comment:
should it not be inside the following if block ? Because it will always be null here.

Reply:
You're right. Fixed, thanks